### ISSUE TITLE
Is Latex supported?

### ISSUE THREAD

**Issue by MenglaiWang** [barryclark/jekyll-now#583]
I wonder whether latex is supported.Or how can I get my bolg support latex?

**Comment by act65** [comment]
Yea, you can get it working with mathjax. Steal some code from mine if you like. You just have to add some lines of code to one of the includes files and into the index... i forget what i did.

Although, the reason I am here is that when someone else forked my blog the mathjax stopped working. Anyone?

**Comment by changkaizhao** [comment]
hi @act65  , After I fork your blog. my page looks like this[my site](https://changkaizhao.github.io/FAT/) ..  mathjax does not work..

**Comment by act65** [comment]
hey, yea. I think there is something wrong with my blog... Although it seems to work fine for me. As mentioned above, mathjax doesnt work when others try to fork it...

> Although, the reason I am here is that when someone else forked my blog the mathjax stopped working. Anyone?

I got mine working by including [this](https://github.com/act65/act65.github.io/blob/master/_includes/mathjax.html) into my default page layout with `{% include mathjax.html %}`.
Good luck...

**Comment by changkaizhao** [comment]
@act65  unluckily it fails.    I fork your project , the mathjax.html is  included in every html file in layouts. I spent several days on this issue....   unluckily no answer at all...

**Comment by changkaizhao** [comment]
@act65  open your blog site failed.   https://act65.github.io

**Comment by changkaizhao** [comment]
@act65  your site is failed to show math correctly on my browser  (safari and chrome  on MAC)...   https://act65.github.io

**Comment by act65** [comment]
hmm. sorry man, I had to battle with it to get it to work as well. To be honest I have no clue what I am doing...


otherwise, try reloading my blog?!? use this link http://act65.github.io/FAT/

![screen shot 2016-09-12 at 4 19 49 pm](https://cloud.githubusercontent.com/assets/6046380/18424529/d721af06-7904-11e6-85eb-8da610ac38cd.png)

**Comment by act65** [comment]
hmm. you still need `{% include meta.html %}` in the default

**Comment by changkaizhao** [comment]
@act65  are you sure your page link is `http://` not `https://` ? without `s` the page could not be opened. with `s` it ok. but math equations do not show correctly.

**Comment by act65** [comment]
Ok, I tried another script (see [here](https://github.com/act65/act65.github.io/blob/master/_includes/mathjax.html)). Does this work for you? https://act65.github.io

**Comment by changkaizhao** [comment]
@act65  yes, I finally figure out what is wrong with it. the `src = https://`  is work for `https://` not `http://`    my browser not work with `http://` I don't know why..    if I use `https://`  the (Disqus) does not work..   I am totally mixed up...

**Comment by changkaizhao** [comment]
@act65  jekyll not redirect `http://` to `https://` automatically.  My browser could not open `http://` ..  what's wrong with it?

**Comment by act65** [comment]
No idea...

**Comment by changkaizhao** [comment]
@act65  I solved this problem ! I brought a custom domain. use my own domain I could uncheck `Enforce HTTPS`  in settings.   everything goes well.      the only problem left is Disqus not show in Safari.

**Comment by jinzhenfan** [comment]
Just want to say thank you! I get the Latex working!
